modules/tensor_mechanics/test/tests/umat/multiple_blocks/tests

```diff
@@ -31,5 +31,30 @@
                   'constitutive models, with the ability to provide different parameters for '
                   'different blocks using distinct interface plug-ins in multi processor runs.'
   []
-
+  [rve_multimaterial_umat]
+    type = 'Exodiff'
+    input = 'rve_multimaterial.i'
+    exodiff = 'rve_multimaterial_umat_out.e'
+    cli_args = "Materials/active='umat_1 umat_2' "
+               "Outputs/file_base=rve_multimaterial_umat_out"
+    library_mode = 'DYNAMIC'
```

Comment:
Need to add this to both tests:

```suggestion
    library_mode = 'DYNAMIC'
```

Reply:
Why to those two tests?

Reply:
Okay, I am reverting those new options since I am getting parse errors for now.

Reply:
It appears it's only needed in the combined module, per @milljm's PR https://github.com/milljm/moose/blob/b66fa8b937baf04917185244806d51a6d84b335b/modules/combined/test/tests/umat/tests#L23